Wireframe/css/style.css

```diff
@@ -32,60 +31,40 @@ p {
     margin: 5px 0;
 }
 
-/* Articles Section */
-.articles {
-    flex: 1;
-    padding: 20px;
-    background-color: #f4f4f4;
-
-}
 
 article {
     background-color: #e0ffe0;
-    margin: 20px 150px;
-    padding: 15px;
-    border: 1px solid #ccc;
-    border-radius: 5px;
-    border-color: #4CAF50;
-
+    padding: 30px 30px;
```

Comment:
Hi @Amir200524,
I will suggest using rem over px for elements that do not have fixed sizes. 

Please see link below for reference

https://www.shecodes.io/athena/28078-when-to-use-ems-rems-and-px-in-css